validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkFeedbackRequestValidatorTest.java

```diff
@@ -174,45 +175,44 @@ public void setUpLocalTests() {
      * Test success flow.
      */
     @Test
-    public void testBulkReferenceFound() {
+    void testBulkReferenceFound() {
         // Tell the mock dao to return this request
-        expect(mockIBulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, reference, dataRetentionPeriod))
-                .andReturn(bulkSubmission);
-        replay(mockIBulkSubmissionDao);
+        when(mockIBulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, REFERENCE, dataRetentionPeriod))
+                .thenReturn(bulkSubmission);
 
         // Inject the mock dao into the validator.
         validator.setBulkSubmissionDao(mockIBulkSubmissionDao);
 
         // Validate the bulk customer.
         bulkFeedbackRequest.accept(validator, null);
-        EasyMock.verify(mockIBulkSubmissionDao);
-
+        verify(mockIBulkSubmissionDao).getBulkSubmissionBySdtRef(bulkCustomer, REFERENCE, dataRetentionPeriod);
     }
 
     /**
      * Test success flow.
      */
     @Test
-    public void testBulkReferenceNotFound() {
+    void testBulkReferenceNotFound() {
+        when(errorMessagesCache.getValue(IErrorMessage.class, IErrorMessage.ErrorCode.BULK_REF_INVALID.name()))
+                .thenReturn(errorMessage);
 
         // Tell the mock dao to return this request
-        expect(mockIBulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, reference, dataRetentionPeriod))
-                .andReturn(null);
-        replay(mockIBulkSubmissionDao);
+        when(mockIBulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, REFERENCE, dataRetentionPeriod))
+                .thenReturn(null);
 
         // Inject the mock dao into the validator.
         validator.setBulkSubmissionDao(mockIBulkSubmissionDao);
 
         try {
             // Validate the request
             bulkFeedbackRequest.accept(validator, null);
-            EasyMock.verify(mockIBulkSubmissionDao);
-            Assert.fail("Failed to throw expected InvalidBulkReferenceException");
+            verify(mockIBulkSubmissionDao).getBulkSubmissionBySdtRef(any(), any(), any());

```

Comment:
Should this verify the specific values rather than using any()?  The equivalent statement on line 188 verifies the specific values.